### PR TITLE
Adds new card for BC registries API Gateway

### PR DESCRIPTION
**Pull request by orelbn**
*Issue #:* bcgov/entity#17308

*Description of changes:*
- Adds new card that links to BC registries API Gateway

### CODE REVIEW

**Comment by orelbn** [comment]
@pwei1018 Do you need to update any of the deployment set up for the link to work correctly? I believe it should just copy it from the .env.example file into .env 

Also can you please assign this Pull request to myself and add the necessary reviewers.

**Comment by pwei1018** [comment]
Temporary Url for review: https://yfthig-dev--pr-142-x1mfzgde.web.app

**Comment by pwei1018** [comment]
> @pwei1018 Do you need to update any of the deployment set up for the link to work correctly? I believe it should just copy it from the .env.example file into .env
> 
> Also can you please assign this Pull request to myself and add the necessary reviewers.

In devops/vaults.env, you need to add 

REGISTRIES_API_GATEWAY_URL="op://web-url/$APP_ENV/bcregistry/REGISTRIES_API_GATEWAY_URL"

So that the CD pipeline will get the value from 1password vault service.

File: .env.example [hunk 1]
```diff
@@ -24,6 +24,7 @@ BCOL_URL="https://d1.bconline.gov.bc.ca/"
 BCEID_URL="https://www.test.bceid.ca/os/?7686&SkipTo=Basic"
 NAME_REQUEST_URL="https://dev.names.bcregistry.gov.bc.ca/"
 ORGBOOK_URL="https://orgbook.gov.bc.ca/"
+REGISTRIES_API_GATEWAY_URL="https://developer.bcregistry.daxiom.ca/"
```
[the remark this answers is not in the document]
Comment (orelbn) [reply .env.example]: Updated thanks!

**Review by pwei1018 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://yfthig-dev--pr-142-x1mfzgde.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://yfthig-dev--pr-142-x1mfzgde.web.app

File: utils/feature-flags.ts [hunk 2]
```diff
@@ -7,14 +7,16 @@ declare const window: any
  * Default flag values when LD is not available.
  */
 const defaultFlagSet: LDFlagSet = {
+  'bcregistry-ui-bca-enabled': false,
   'bcregistry-ui-bus-search-enabled': true,
   'bcregistry-ui-bus-search-beta-chip': false,
   'bcregistry-ui-bus-search-coming-soon-chip': false,
+  'bcregistry-ui-cso-enabled': false,
```
Comment (seeker25) [inline utils/feature-flags.ts:14]: Was this tested?

File: utils/feature-flags.ts [hunk 3]
```diff
@@ -7,14 +7,16 @@ declare const window: any
...
+  'bcregistry-ui-esra-enabled': false,
```
Comment (seeker25) [inline utils/feature-flags.ts:15]: This shouldn't be false, it should be true.

File: utils/feature-flags.ts [hunk 4]
```diff
@@ -7,14 +7,16 @@ declare const window: any
...
   'bcregistry-ui-mhr-enabled': false,
```
Comment (seeker25) [inline utils/feature-flags.ts:16]: Probably should be true as well

File: utils/feature-flags.ts [hunk 5]
```diff
@@ -7,14 +7,16 @@ declare const window: any
  * Default flag values when LD is not available.
  */
 const defaultFlagSet: LDFlagSet = {
+  'bcregistry-ui-bca-enabled': false,
```
Comment (seeker25) [inline utils/feature-flags.ts:10]: not launched yet. looks correct

**Comment by orelbn** [comment]
> 

Yes I can set the flags based on your comments above. 
Regarding your comments about the testing, I am not sure what its referring to please me on rocketChat

**Comment by seeker25** [comment]
image

**Comment by seeker25** [comment]
creating a new PR, should cover this

**Pull request merged**